pennylane/templates/subroutines/select.py

```diff
@@ -98,6 +99,21 @@ def __init__(self, ops, control, id=None):
         all_wires = target_wires + control
         super().__init__(*self.data, wires=all_wires, id=id)
 
+    def __copy__(self):
+        """Copy this op"""
+        cls = self.__class__
+        copied_op = cls.__new__(cls)
+
+        new_data = copy.copy(self.data)
+
+        for attr, value in vars(self).items():
```

Comment:
Not related to this PR, but why do we use `hyperparameters` in this template instead of `_hyperparameters`? I thought the convention is to use the latter and then access it via this [property](https://github.com/PennyLaneAI/pennylane/blob/2ca439887145a61d068bbf4e0e282a16a4cfe2a8/pennylane/operation.py#L1199).

Reply:
Also happens for another template, will address in a new PR!